firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -1050,6 +1050,93 @@ extension String {
     }
 }
 
+// MARK: - Micro survey
+extension String {
+    public struct Microsurvey {
+        public struct Prompt {
+            public static let TitleLabel = MZLocalizedString(
+                key: "Microsurvey.Prompt.TitleLabel.v127",
+                tableName: "Microsurvey",
+                value: "Help us make Firefox better. It only takes a minute.",
```

Comment:
Firefox should be a placeholder `%@`, and then you need to interpolate the string with `AppName.shortName.rawValue`

Reply:
ah missed this, thanks for the catch!